services/docker-compose.yml

```diff
@@ -1440,12 +1386,7 @@ services:
     init: true
     hostname: "{{.Node.Hostname}}-{{.Task.Slot}}"
     environment:
-      POSTGRES_DB: ${POSTGRES_DB}
-      POSTGRES_ENDPOINT: ${POSTGRES_ENDPOINT}
-      POSTGRES_HOST: ${POSTGRES_HOST}
-      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
-      POSTGRES_PORT: ${POSTGRES_PORT}
-      POSTGRES_USER: ${POSTGRES_USER}
+      <<: *postgres_settings
```

Comment:
For the migration I would have a fixed lower min/max setup. What do you think? Perhaps we can even shutdown the service after it runs.

Reply:
but the migration does not check min and max does it?